src/ImageSharp/Formats/Jpg/Components/Block8x8F.Generated.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+
+namespace ImageSharp.Formats
+{
+	internal partial struct Block8x8F
```

Comment:
✈️ VROOOOOOOOOM......

That's the sound of all this going over my head. Could you please add some comments here so I can figure out what is going on? It's proper into CS territory here.